### PR TITLE
List

### PR DESCRIPTION
**Pull request by anandsr21harshit**
Simple and Stacked List

### CODE REVIEW

File: Components/List/list.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
+.list-container{
+    list-style-type: none;
+    border: 1px solid #e6e6e9;
```
Comment (OmKakatkar) [inline Components/List/list.css]: Try to use CSS variables for colors

File: Components/List/list.css [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    font-size: 1.5rem;
+    margin-bottom: 1rem;
+}
+
+.list-container >li{
+    margin: 0.8rem 0;
+}
+
+.stacked{
+    background-color: rgb(194, 191, 191);
```
Comment (OmKakatkar) [inline Components/List/list.css]: Try using CSS variables here as well.

**Review by OmKakatkar — CHANGES_REQUESTED** [review]

**Review by shrey27 — APPROVED** [review]

File: Components/List/list.css (hunk 1 above)
Comment (anandsr21harshit) [reply Components/List/list.css]: Done

File: Components/List/list.css (hunk 2 above)
Comment (anandsr21harshit) [reply Components/List/list.css]: Done

**Review by OmKakatkar — APPROVED** [review]

**Pull request merged**